### ISSUE TITLE
Report cronjobs workload

### ISSUE THREAD

**Issue by gaetanars** [snyk/kubernetes-monitor#654]
Is it possible to scan cronjobs workload with Snyk monitor?

Only Daemonset, Deployment or Statefulset was send in our snyk Kubernetes integration.

**Comment by ivanstanev** [comment]
Hey @gaetanars 👋 

We currently do support CronJobs, we support all the native Kubernetes workloads. However, right now we support the `batch/v1beta1` API for CronJobs, as noted in this comment in the source: https://github.com/snyk/kubernetes-monitor/blob/f1b451bfd007a6764ace7a572b595bafd88a571e/src/supervisor/types.ts#L41-L43

Are you using CronJobs from a different Kubernetes API? Perhaps batch/v1 or batch/v2 if it's now released?

**Comment by gaetanars** [comment]
Dear @ivanstanev,

I use `batch/v1beta1` in Kubernetes 1.19. It seems to work only with annotations.

I have installed snyk-monitor without modify any of our workloads, I can see deployment, statefulset or daemonset in the Snyk interface when I want to import project form Kubernetes integration, but I don't see CronJobs.

Yesterday I have annotated one of our CronJobs and it was automatically imported in Snyk.

I close this issue, tahnks
